sources/Makefile

```diff
@@ -68,8 +68,8 @@ export CRA_PROGRAM_AREA_CODE := $(or $(CRA_PROGRAM_AREA_CODE), )
 export CRA_ENVIRONMENT_CODE := $(or $(CRA_ENVIRONMENT_CODE), )
 # Versions used by camunda-docker compose-core.yml during the make camunda.
 # Versions upgraded following the conbination available on https://github.com/camunda/camunda-platform/blob/main/.env
-export CAMUNDA_CONNECTORS_VERSION := $(or $(CAMUNDA_CONNECTORS_VERSION), 8.5.3)
-export CAMUNDA_PLATFORM_VERSION := $(or $(CAMUNDA_PLATFORM_VERSION), 8.5.2)
+export CAMUNDA_CONNECTORS_VERSION := $(or $(CAMUNDA_CONNECTORS_VERSION), 8.6.9)
+export CAMUNDA_PLATFORM_VERSION := $(or $(CAMUNDA_PLATFORM_VERSION), 8.6.12)
```

Comment:
The `ELASTIC_VERSION` can be updated to at least `8.17.1` that is used in the https://github.com/camunda/camunda-platform/blob/main/.env for version `8.6.8` of the `CAMUNDA_PLATFORM_VERSION`.